assets/js/phoenix_live_view/constants.js

```diff
@@ -2,7 +2,12 @@ export const CONSECUTIVE_RELOADS = "consecutive-reloads"
 export const MAX_RELOADS = 10
 export const RELOAD_JITTER_MIN = 5000
 export const RELOAD_JITTER_MAX = 10000
-export const FAILSAFE_JITTER = 30000
+export const RELOAD_FAILSAFE_JITTER = 30000
+export const CONSECUTIVE_REDIRECTS = "consecutive-redirects"
+export const MAX_REDIRECTS = 3
+export const REDIRECT_JITTER_MIN = 1000
+export const REDIRECT_JITTER_MAX = 5000
+export const REDIRECT_FAILSAFE_JITTER = 20000
```

Comment:
I could have re-used the `RELOAD_` constants but opted to make new ones specifically for redirects.